spec/api-net-spec.ts

```diff
@@ -1417,7 +1415,6 @@ describe('net module', () => {
           if (details.url === `${serverUrl}${requestUrl}`) {
             requestIsIntercepted = true;
             // Disabled due to false positive in StandardJS
```

Comment:
Let's also delete these lines in this file since they aren't applicable once the eslint comment below it is removed.